data/json/construction.json

```diff
@@ -2100,16 +2100,18 @@
         "category" : "OTHER",
         "required_skills" : [ [ "survival", 0 ] ],
         "time" : 6,
-        "qualities": [ [ { "id": "CUT", "level": 1 } ] ],
         "pre_terrain" : "t_grass_long",
-        "post_terrain" : "t_grass"
+        "post_terrain" : "t_grass",
+        "tools": [
+        [ [ "scythe", 2 ], [ "sickle", 2 ] ]
```

Comment:
Remove `2`s, as it would try to consume 2 charges of scythe or sickle, and it's good to go.